testing/trino-product-tests-launcher/src/main/resources/docker/presto-product-tests/common/hadoop-kerberos/krb5.conf

```diff
@@ -0,0 +1,25 @@
+# Copy of hdp2.6-hive-kerberized/etc/krb5.conf with lower ticket_lifetime
+[logging]
+ default = FILE:/var/log/krb5libs.log
+ kdc = FILE:/var/log/krb5kdc.log
+ admin_server = FILE:/var/log/kadmind.log
+
+[libdefaults]
+ default_realm = LABS.TERADATA.COM
+ dns_lookup_realm = false
+ dns_lookup_kdc = false
+ forwardable = true
+ allow_weak_crypto = true
+ # low ticket_lifetime to make sure refresh code in Trino is tested
+ ticket_lifetime = 30s
```

Comment:
Note that I had to remove `renew_lifetime` from here to avoid hitting https://github.com/trinodb/trino/issues/11251#issuecomment-1061566447.

We can add it back if we have https://github.com/trinodb/docker-images/pull/162 but I don't see any reason to have it since our code doesn't care whether tickets are renewable or not since it always performs a "relogin" instead of renewal.

Reply:
reminder to self, https://github.com/trinodb/docker-images/pull/162 is moving forward.